### PR TITLE
WIP: Add an Example module

### PR DESCRIPTION
**Pull request by amitaibu**
To get #2 in place, I have started adding an example module.

It doesn't work yet. In fact I still get an error that implies I'm probably declaring incorrectly the module in the elm-package. But it's probably a good start :)

Would love pointers in case I'm missing obvious stuff.

__src_simple_elm_and___examples_07-websockets_elm

I have deliberately done it in a single file for now, just to make it a little easier the grok it.

I think the next thing to do after getting this to compile, is have some server side code to emit two different server-sent events and see how the app handles them.

### CODE REVIEW

File: example/src/Simple.elm [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+
+-- SUBSCRIPTIONS
+
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    model.channels
+        |> Dict.filter (\_ channel -> channel.listening)
+        |> Dict.map (\key val -> (listen model.eventRoot key))
```
Comment (amitaibu) [inline example/src/Simple.elm:123]: The code in the README says `|> Dict.map (\key val -> Rest.listen model.eventRoot key val.typ)` but this `val.typ` isn't clear.

**Comment by amitaibu** [comment]
btw, regarding the error message - seems that with `elm-reactor` going to `http://localhost:8000/src/EventSource.elm` causes an error also on `master`

**Comment by amitaibu** [comment]
Note that the example wasn't working yet, due to runtime errors.

**Comment by martintrojer** [comment]
Yeah, np. I can have a look in a while.

Great start tho! 👍

**Pull request merged**